src/core/transport/static_metadata.h

```diff
@@ -33,11 +33,11 @@
  * WARNING: Auto-generated code.
  *
  * To make changes to this file, change
- * tools/codegen/core/gen_static_metadata.py,
+ *tools/codegen/core/gen_static_metadata.py,
```

Comment:
Is there a bug where the generated code here is dropping a space? If so, maybe file an issue?